client/app/components/app-header/components/FavoritesDropdown.jsx

```diff
@@ -27,7 +27,7 @@ export default function FavoritesDropdown({ fetch, urlTemplate }) {
   }, [fetch]);
 
   // fetch items on init
-  useEffect(() => fetchItems(false), []);
+  useEffect(() => fetchItems(false), [fetchItems]);
```

Comment:
Are you sure this should be fixed? According to comment, it should fetch items only once, but with this fix it will re-load items each time `fetchItems` changed

Reply:
Not sure at all - it's part of the automatic fixes.

Reply:
I'm also not sure how it supposed to work. Maybe @gabrieldutra will help :thinking:

Reply:
I'm testing these eslint changes, it's indeed a case where it makes it mandatory. In this specific case I think it's ok to keep the dependency as `fetchItems` should only change if the `fetch` prop change (in this case I think we do want the items to be loaded again).

Though from a quick search there are discussions around this (like [this](https://github.com/facebook/create-react-app/issues/6880) one). Just need to confirm if there's a more correct React way (they do mention `useMemo`, but imo that only applies to when you don't have a Promise to solve).

A side note I just noticed: I'd avoid the `() => executeFunction()` for `useEffect` as the return value is used for cleaning. Anyway, I'll check other replacements to see if they can be problematic.